colorbleed/plugins/maya/publish/validate_model_content.py

```diff
@@ -63,7 +63,8 @@ def get_invalid(cls, instance):
             cls.log.error("Must have exactly one top group")
             if len(assemblies) == 0:
                 cls.log.warning("No top group found. "
-                                "(Are there objects in the instance?)")
+                                "(Are there objects in the instance?"
+                                " Or is it parented in another group?)")
```

Comment:
This extra line in the warning message is to guide the artist more whenever the node is parented in a group that is not being published along.